### PR TITLE
Update manual upgrade page with better order of operations

### PR DESCRIPTION
**Pull request by dereknola**
Addresses: https://github.com/k3s-io/docs/issues/178

~~Mentions that you have to stop the existing K3s service before running the installation script with a new version.~~

### CODE REVIEW

**Comment by brandond** [comment]
> have to stop the existing K3s service before running the installation script with a new version

Do you though? I thought that the install script would restart the service for you if the config or version changed.

**Comment by dereknola** [comment]
Ah I guess your right. My brain was just on autopilot when looking at the install script output:

```
[INFO]  Creating killall script /usr/local/bin/k3s-killall.sh
[INFO]  Creating uninstall script /usr/local/bin/k3s-uninstall.sh
[INFO]  env: Creating environment file /etc/systemd/system/k3s.service.env
[INFO]  systemd: Creating service file /etc/systemd/system/k3s.service
[INFO]  systemd: Enabling k3s unit
Created symlink /etc/systemd/system/multi-user.target.wants/k3s.service → /etc/systemd/system/k3s.service.
[INFO]  systemd: Starting k3s

```

We weirdly don't have output that the old k3s service was stopped.

**Comment by brandond** [comment]
It's because we just log that we start it when actually we restart it. Which is either a start or a stop/start depending on current state.
https://github.com/k3s-io/k3s/blob/master/install.sh#L1042-L1045

File: docs/upgrades/manual.md [hunk 1]
```diff
@@ -37,6 +37,12 @@ The contents of the [configuration file](../installation/configuration.md#config
 If you want your configuration to be independent from the install script, you should use a configuration file instead of passing environment variables or arguments to the install script.
 :::
 
+Running the install script will:
+
+1. Download the new k3s binary
+2. Stop the existing k3s service
+3. Start the new k3s service
```
Comment (brandond) [inline docs/upgrades/manual.md:44]:
```suggestion
2. Update the systemd unit or openrc init script to reflect the args passed to the install script
3. Restart the k3s service
```

**Review by brandond — CHANGES_REQUESTED** [review]

**Review by brandond — APPROVED** [review]

**Review by vitorsavian — APPROVED** [review]

**Review by cwayne18 — APPROVED** [review]

**Pull request merged**